code/clue.js

```diff
@@ -10,50 +9,202 @@ const mrGreen = {
   occupation: 'Entrepreneur'
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'He is very quiet',
+  age: 39,
+  image: 'assets/plum.png',
+  occupation: 'Professor'
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She is very smart',
+  age: 37,
+  image: 'assets/scarlet.png',
+  occupation: 'Lawyer'
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She is very rich',
+  age: 58,
+  image: 'assets/peacock.png',
+  occupation: 'Doctor'
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'He is very loyal',
+  age: 62,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel'
+}
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'She is very kind',
+  age: 65,
+  image: 'assets/white.png',
+  occupation: 'Actress'
+}
 
 const rope = {
   name: 'rope',
-  weight: 10
+  weight: 10,
+  color: 'black',
+  image: 'assets/rope.png'
+}
+
+const knife = {
+  name: 'knife',
+  weight: 4,
+  color: 'black',
+  image: 'assets/knife.png'
+}
+
+const candlestick = {
+  name: 'candestick',
+  weight: 8,
+  color: 'black',
+  image: 'assets/candlestick.png',
+}
+
+const dumbbell = {
+  name: 'dumbbell',
+  weight: 14,
+  color: 'black',
+  image: 'assets/dumbbell.png'
+}
+
+const poison = {
+  name: 'poison',
+  weight: 1,
+  color: 'black',
+  image: 'assets/poison.png'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const axe = {
+  name: 'axe',
+  weight: 12,
+  color: 'black',
+  image: 'assets/axe.png'
```

Comment:
Adding pictures to the weapons deck is a great addition, wish I had thought of it myself, haha!